src/parser/rogue/assassination/CHANGELOG.js

```diff
@@ -1,11 +1,13 @@
 import React from 'react';
 
-import { tsabo, Cloake, Zerotorescue, Gebuz, Aelexe, Vetyst, joshinator } from 'CONTRIBUTORS';
+import { tsabo, Cloake, Zerotorescue, Gebuz, Aelexe, Vetyst, joshinator, Tyndi } from 'CONTRIBUTORS';
 import SPELLS from 'common/SPELLS';
 import SpellLink from 'common/SpellLink';
 import { change, date } from 'common/changelog';
 
 export default [
+  change(date(2020, 10, 2), <>Added Dashing Scoundrel Legendary</>, [Tyndi]),
+  change(date(2020, 9, 29), <>Updated for Shadowlands</>, [Tyndi]),
```

Comment:
Remove the Fragments if you arent using them

Alternatively you could include the SpellLink for the legendary <SpellLink id={SPELLS.DASHING_SCOUNDREL.id} />